### PR TITLE
build: update Dockerfile to track debian-slim

### PR DESCRIPTION
**Pull request by vegano1**
[RET-1177](https://opentrons.atlassian.net/browse/RET-1177)

Debian 10 has reached EOL, update to debian-slim to track the latest stable (currently Debian 11)

[RET-1177]: https://opentrons.atlassian.net/browse/RET-1177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
nice

File: Dockerfile
Comment (SyntaxColoring) [inline Dockerfile]: To reduce unpleasant surprises in the future, I would consider keeping this pinned to the major version:

```suggestion
FROM debian:11-slim
```

(from https://hub.docker.com/_/debian)

From what I remember from working on #132, when `stable-slim` rolled over from `10-slim` to `11-slim`, it caused downstream problems that weren't trivial to debug.

**Review by SyntaxColoring — COMMENTED** [review]
I guess this needs to wait until after #166 is merged, right?

**Review by SyntaxColoring — APPROVED** [review]
Thanks!

**Comment by vegano1** [comment]
Was merged to edge

**Comment by SyntaxColoring** [comment]
@vegano1 Are you sure this can be closed? We're still using `10-slim` in the `opentrons-develop` branch.

**Pull request closed without merging**
